piksi_tools/interpolate_event_positions.py

```diff
@@ -0,0 +1,150 @@
+"""
+interpolate_event_trigger.py allows user to analyze log files
+with external event triggers.
+"""
+
+from sbp.client.loggers.json_logger import JSONLogIterator
+from sbp.ext_events import SBP_MSG_EXT_EVENT
+import csv
+
+def lin_interp(o, n, otow, ntow, ttow):
+  """
+  Linearly interpolates distance.
+  Interpolation accurate to the mm.
+  
+  Parameters
+  ----------
+  o : integer
+    distance previous to trigger
+  n : integer
+    distance prior to trigger
+  otow : integer
+    TOW of data packet previous to trigger
+  ntow : integer
+    TOW of data packet after trigger
+  ttow : integer
+    TOW of trigger 
+  """
+  d = float(n-o)
+  t = (ntow-otow) 
```

Comment:
You need to check the assumptions here about the order of otow, ntow, and tow.  You can consider doing an "assert" statement that otow < ttow < ntow.  This shouldn't happen but is worth checking.  Also might be worth checking against a maximum interpolation time and warning the user.  For instance, maybe we should just error out if the ntow - otow is greater than a few seconds.